homeassistant/components/modbus/modbus.py

```diff
@@ -249,17 +249,22 @@ async def async_setup(self):
         for entry in self._pb_call.values():
             entry[ENTRY_FUNC] = getattr(self._client, entry[ENTRY_NAME])
 
+        await self.async_connect_task(None)
+        return True
+
+    async def async_connect_task(self, args):
```

Comment:
If we don't schedule this with the event listener we don't need to accept a parameter anymore.

Reply:
Removed